src/pages/Content/index.js

```diff
@@ -13,3 +13,9 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
+function playRecording(recordedEvents) {
+  recordedEvents.forEach(element => {
+    document.querySelector(element.selector).eval(element.type)();
```

Comment:
Add a `switch` or `if/else` to check what action to produce, don't use `eval`.

Right now we only have a `click` action so you only need to check for that type and click it via the `click` function created above.

Reply:
Yul has updated the code in new change